### PR TITLE
Add live radio to bbc a11y checks

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Add live radio to bbc a11y checks

**Code changes:**

- Add live radio to bbc a11y checks

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by dr3** [comment]
Failing due to
```
✗ http://localhost.bbc.com:7080/afaanoromoo/bbc_afaanoromoo_radio/liveradio --width 360

  * Editorial: Indicating language: Html must have lang attribute
    - In frame //div[@id='root']/main/div/div/div/div/iframe : html tag has no lang attribute: /html
  * Structure: Headings: Exactly one main heading
    - In frame //div[@id='root']/main/div/div/div/div/iframe : Found 0 h1 elements.
  * Structure: Containers and landmarks: Exactly one main landmark
    - In frame //div[@id='root']/main/div/div/div/div/iframe : Found 0 elements with role="main".
```

File: a11y.js [hunk 1]
```diff
@@ -11,38 +11,39 @@ const baseUrl = 'http://localhost.bbc.com:7080';
...
-
-urls.forEach(url =>
-  pageWidths.forEach(width =>
-    // eslint-disable-next-line no-undef
-    page(url, { width, hide }),
-  ),
-);
+
+// '//div[@id='root']/main/div/div/div/div/iframe' Added to hide
+// iframe errors to be fixed inhttps://github.com/bbc/bbc-a11y/issues/298
```
Comment (dr3) [inline a11y.js]:
```suggestion
// iframe errors to be fixed in https://github.com/bbc/bbc-a11y/issues/298
```

File: a11y.js [hunk 2]
```diff
@@ -11,38 +11,39 @@ const baseUrl = 'http://localhost.bbc.com:7080';
...
+
+const pageTypes = {
+  frontPage: ['/html/head/iframe'],
+  articles: ['/html/head/iframe'],
+  liveRadio: [
+    '/html/head/iframe',
+    "//div[@id='root']/main/div/div/div/div/iframe",
```
Comment (pjlee11) [inline a11y.js:38]: 🤢

**Review by pjlee11 — APPROVED** [review]
Makes sense

**Review by ibMadbouly — APPROVED** [review]
LGTM 👍

**Review by ryanmccombe — APPROVED** [review]

**Pull request merged**